frontend/src/static/js/v_authorship.js

```diff
@@ -88,18 +89,36 @@ window.vAuthorship = {
 
     getRepoProps(repo) {
       if (repo) {
-        const author = repo.users.filter((user) => user.name === this.info.author);
-        if (author.length > 0) {
-          this.info.name = author[0].displayName;
-          this.filesLinesObj = author[0].fileTypeContribution;
+        if (this.info.isMergeGroup) {
+          // sum of all users' file type contribution
+          repo.users.forEach((author) => {
+            this.updateTotalFileTypeContribution(author.fileTypeContribution);
+          });
+        } else {
+          const author = repo.users.find((user) => user.name === this.info.author);
+          if (author.length > 0) {
```

Comment:
Is `author` a single value now, if we are using `find`? Would that mean we do not need to check the length?

Reply:
Yes author is a single value, we still need to check if `length > 0` just in case

Reply:
I see. Just to clarify, if there are no matches, `find` would return undefined? In that case, there wouldn't be a length property?

Reply:
ahhh i see, good catch, thanks for the investigation

Reply:
Also, within the `if` branch, should we be using `author.displayName` instead of `author[0].displayName` ?